### PR TITLE
Profile builder

### PR DESCRIPTION
**Pull request by scooter72**

### CODE REVIEW

File: JWT/src/main/java/io/scalecube/security/Profile.java [hunk 1]
```diff
@@ -25,12 +25,12 @@
    * @param givenName of the user.
    * @param claims claims as key values.
    */
-  public Profile(String userId, String tenant, String email, Boolean isEmaildVerified, String name,
+  private Profile(String userId, String tenant, String email, Boolean isEmaildVerified, String name,
```
Comment (aharonha) [inline JWT/src/main/java/io/scalecube/security/Profile.java]: `d` should be removed from `isEmaildVerified`

Comment (aharonha) [reply JWT/src/main/java/io/scalecube/security/Profile.java]: Create a constructor with builder as a parameter (avoiding this parameter mess)
``` for (field: this.class.fields) {
code.append (this.field = builder.field);
} ```

**Comment by dmytro-lazebnyi** [comment]
It is duplication (related to Profile.Builder) of pull request #35, isn't it?

**Comment by dmytro-lazebnyi** [comment]
It is duplication (related to Profile.Builder) of pull request #35, isn't it? @scooter72

**Pull request closed without merging**